### PR TITLE
feat(backend): add addOffer and removeOffer backend to blockchain route

### LINKED ISSUE #763: UI for donor users to easily redeem tokens for an offer in the offer's page
Backend: @leisterfrancisco 
Front-end: @Edgar097

### PR DESCRIPTION
**Pull request by leisterfrancisco**
Resolves #763 

### What does this PR do?
Add addOffer and removeOffer backend connection to blockchain

### How should this be tested?
- Make run
- Register a new offer
- Remove offer

### CODE REVIEW

**Comment by leisterfrancisco** [comment]
@Edgar097 please, take a look at merge conflicts

**Review by steph222 — CHANGES_REQUESTED** [review]
Fix merge conflicts

**Comment by Edgar097** [comment]
Check if the conflicts are solved properly since is work done by you @leisterfrancisco

**Review by leisterfrancisco — COMMENTED** [review]
> Check if the conflicts were solved properly since is work done by you @leisterfrancisco

@Edgar097 I did not remember it was mine, or if not, I would have solved it, so sorry

File: hapi/src/utils/lifebankcode.js [hunk 1]
```diff
@@ -220,6 +220,57 @@ const getUserNetworks = async (user) => {
...
+        ],
+        account: CONTRACT_NAME,
+        name: 'addoffer',
+        data: {
+          offer_name: offer.offer_name,
+          sponsor_name: account,
+          category: offer.offer_type,
+          beginning_date: offer.start_date,
+          ending_date: offer.end_date,
+          cost: '10 LIFE',
```
Comment (leisterfrancisco) [inline hapi/src/utils/lifebankcode.js:241]: Change it to real offer cost

File: hasura/metadata/actions.graphql [hunk 2]
```diff
@@ -66,6 +66,25 @@ type Mutation {
...
+    images: String!
+    sponsor_id: Int!
+    active: Boolean!
+    offer_name: String!
+    cost_in_tokens: Int!
+    icon: String!
+  ): create_offer_output
+}
+
+
```
Comment (steph222) [inline hasura/metadata/actions.graphql:87]: Remove space

Comment (leisterfrancisco) [reply hasura/metadata/actions.graphql:87]: @steph222 we didn't add that double empty space, that is the Prettier tool I think

**Review by steph222 — APPROVED** [review]

**Pull request merged**